src/pattern.rs

```diff
@@ -206,40 +206,31 @@ impl BasePattern {
         }
     }
 
-    /// Check if the `identifier` is contained inside the pattern.
-    pub fn contains(&self, identifier: &Identifier) -> bool {
```

Comment:
In 25f810963b58cff14a7306edbd744b3c1d7595e8:

Might as well keep this method, though it can just be a oneliner `self.get(identifier).is_some()`.